docs/installation.md

```diff
@@ -31,6 +31,12 @@ Build across all packages
 yarn run build
 ```
 
+Or if you are using [Lerna](https://lerna.js.org/):
+
+```bash
+lerna bootstrap
+```
+
```

Comment:
not sure we need this section. Since we are using workspaces, yarn install should always suffice. We should remove `yarn run build` section since install will build code as well